### PR TITLE
implement keeper policy functions and bidding models

### PR DESCRIPTION
**Pull request by sehyunc**
* add keeper_bid and keeper_bid_generator to `policies.py`
* create new `models.py` to hold bidding models
* assumed that state for keepers would change to object with` keeper id` as key and `bidding model` value

### CODE REVIEW

**Comment by sehyunc** [comment]
dang my bad for the one lint commit, didn't realize  my other commit messages get straight up rejected

File: dai_cadcad/policies.py [hunk 1]
```diff
@@ -439,6 +440,41 @@ def flipper_deal(flipper, vat, cat, bid_id, now):
 # ---
 
 
+# Keeper
+
+
+def keeper_bid(user_id, model, flipper, vat):
```
Comment (akirillo) [inline dai_cadcad/policies.py:446]: Have this take in `now` and `bid_id` as arguments

File: dai_cadcad/policies.py [hunk 2]
```diff
@@ -439,6 +440,41 @@ def flipper_deal(flipper, vat, cat, bid_id, now):
...
+
+def keeper_bid_generator(_params, _substep, _state_hist, state):
+    """ Executes all `keeper_bid` policies for a timestep.
+    """
+
+    bid_buffer = state["flipper_eth"]["bids"]
+    # assuming state is {1: "simple", 2, "medium", 3: "complex"}
+    keepers = state["keepers"]
+    vat = state["vat"]
+    for auction in bid_buffer:
```
Comment (akirillo) [inline dai_cadcad/policies.py:466]: Have this be a `for i in range(len(bid_buffer))`, `i` will actually be the `bid_id` you pass into `keeper_bid`.
You can grab the bid using `bid_buffer[i]`.

File: dai_cadcad/policies.py [hunk 3]
```diff
@@ -439,6 +440,41 @@ def flipper_deal(flipper, vat, cat, bid_id, now):
...
+    bid = model(std_input)
+    flipper_tend(flipper, vat, bid_id, user_id, flipper.lot, bid, flipper.now)
+    return {}
+
+
+def keeper_bid_generator(_params, _substep, _state_hist, state):
+    """ Executes all `keeper_bid` policies for a timestep.
+    """
+
+    bid_buffer = state["flipper_eth"]["bids"]
```
Comment (akirillo) [inline dai_cadcad/policies.py:462]: pull out `flipper = state["flipper_eth"]` first

File: dai_cadcad/policies.py [hunk 4]
```diff
@@ -439,6 +440,41 @@ def flipper_deal(flipper, vat, cat, bid_id, now):
...
+    # assuming state is {1: "simple", 2, "medium", 3: "complex"}
```
Comment (akirillo) [inline dai_cadcad/policies.py:463]: This looks great, go ahead!

File: dai_cadcad/policies.py (hunk 1 above)
Comment (akirillo) [reply dai_cadcad/policies.py:446]: Rename this and `keeper_bid_generator` to be `keeper_flipper_eth_bid`...

**Pull request merged**